Tools-Override/tests.targets

```diff
@@ -198,6 +198,13 @@
       Overwrite="true"
       Encoding="Ascii" />
 
+      <!--======================================================
+                      Section for netfx test runs
+            ====================================================== -->
+    <ItemGroup Condition="'$(BuildingNETFxVertical)' == 'true'" >
+      <TestCommandLines Include="set DEVPATH=%RUNTIME_PATH%" />
```

Comment:
is this means we are not already having setlocal today in the batch file?

---
In reply to: [110053147](https://github.com/dotnet/corefx/pull/17975#discussion_r110053147) [](ancestors = 110053147)